src/azure-cli-core/azure/cli/core/commands/cache.py

```diff
@@ -0,0 +1,131 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+
+import os
+import timeit
+
+from knack.log import get_logger
+from knack.config import get_config_parser
+
+from azure.cli.core.extension import get_extensions
+from azure.cli.core._environment import get_config_dir
+
+logger = get_logger(__name__)
+
+
+CACHE_CONFIG_FILE = os.path.join(get_config_dir(), 'command_module_index.config')
```

Comment:
Why are you using a config file for this? The test index uses a JSON file, which has more flexibility.